base/PerfOptions.php

```diff
@@ -292,6 +295,10 @@ public function __construct(Vector<string> $argv) {
 
     $argTempDir = $this->getNullableString('temp-dir');
 
+    if(array_key_exists('db-host', $o)){
```

Comment:
This needs to be an option that requires `--i-am-not-benchmarking`.  Use one of the `getNullableString` functions to make it so.